.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: ruby
 rvm:
+  - 2.4.0
```

Comment:
Maybe 2.4.1, since that's the latest in the 2.4.x line? And while you're in there... would you be against upping the others to their latest point release in their line?